### DISCUSSION TITLE
COMO PUEDO INSTALAR HANGOVER EN DEBIAN ARM64

### DISCUSSION THREAD

**Discussion by Cristiankon** [AndreRH/hangover#146] [Q&A]
Hola, es posible que puedan explicarme como puedo instalar y ejecutar hangover en debian 12 arm64 debido a que no entiendo como es el proceso para instalar y ejecutar

**Comment by AndreRH** [comment]
Hola, utilice la descarga en "Releases". Descomprime el archivo e instala los componentes siguiendo las instrucciones de tu distribución. Normalmente haciendo doble clic o así:

`sudo apt install ./hangover-libwow64fex_VER_arm64.deb ./hangover-libqemu_VER~bookworm_arm64.deb ./hangover-wine_VER~bookworm_arm64.deb`

Sustituya "VER" por la versión utilizada

**Reply by Cristiankon** [reply]
muchas gracias por aclarar amigo, lo que pasa es que soy nuevo en debian y no la había utilizado antes estoy aprendiendo apenas como  instalar los programas. ¿en que parte esta el instructivo completo para hacerlo paso a paso? te agradeceria bastante.

**Reply by AndreRH** [reply]
Tal vez eso ayude: 
https://es.wikihow.com/instalar-archivos-DEB

**Reply by Cristiankon** [reply]
Hola de nuevo¡, espero te encuentres bien. te escribo para hacerte una consulta resulta que estoy interesado en instalar o ejecutar con hangover un programa de edición de PDF que se llama Gear PDF https://www.pdfgear.com/es/  sin embargo, se me ha dificultado instalar debido que no se ejecuta el código y da error.  puedes explicarme qué pasa y si tiene alguna solución u otra metodología de activación? mi computadora es una chromebook duet 5 arm. gracias por tu atencion.

**Reply by AndreRH** [reply]
Tuve éxito con pdfgear 1.0.0 instalador, pero se instala la versión de 64 bits que no es compatible.

**Comment by Cristiankon** [comment]
Sabes por que las nuevas versiones no se ejecutan? hay algo que se pueda hacer? yo siempre uso el comando de ejecución: wine program.exe

instale la version 1.0.0 pero resulta que tiene que descargar recursos de un servidor pero da un error 404. entonces no ejecuta

**Reply by AndreRH** [reply]
si quieres probar 1.0.0 sin 404, entonces necesitas instalar wine-mono. Puede descargarlo aquí: https://dl.winehq.org/wine/wine-mono/ y esta tabla muestra qué versión: https://wiki.winehq.org/Mono#Versions. 
Luego instala: `wine msiexec /i /ruta/para/tu/version-de-vino-mono-x86.msi`
Después de eso no hay más 404.
